flocker/ca/_ca.py

```diff
@@ -4,6 +4,25 @@
 Low-level logic for a certificate authority.
 
 Uses RSA 4096-bit + SHA 256.
```

Comment:
Not related to this branch, but this seems like overly specific information to offer here.